chocolatey/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,34 @@
+﻿try {
+    $binRoot = Get-BinRoot
+    $poshgitPath = join-path $binRoot 'poshgit'
+
+    try {
+      if (test-path($poshgitPath)) {
+        Write-Host "Attempting to remove `'$poshgitPath`'."
+        remove-item $poshgitPath -recurse -force
+      }
+    } catch {
+      Write-Host "Could not remove `'$poshgitPath`'"
+    }
+
+    if(Test-Path $PROFILE) {
+        $oldProfile = @(Get-Content $PROFILE)
+        $newProfile = @()
+
+        foreach($line in $oldProfile) {
+            if ($line -like '*PoshGitPrompt*') { continue; }
+            if ($line -like '*Load posh-git example profile*') { continue; }
+            if ($line -like '*posh-git*profile.example.ps1*') { continue; }
+
+            $newProfile += $line
+        }
+        Set-Content -path $profile -value $newProfile -Force

```

Comment:
Set-Content defaults to ASCII.  Either we should detect the current encoding like in the other install.ps1 or maybe use `-Encoding Utf8`?  But that will add the UTF-8 BOM.  File encodings are such a PITA.